### PR TITLE
CLDR-15253 Unicode 15 new scripts & script metadata

### PR DESCRIPTION
**Pull request by markusicu**
CLDR-15253

- [ ] This PR completes the ticket.

Updated according to https://cldr.unicode.org/development/updating-codes/updating-script-metadata
For comparison, see PR #1291 for Unicode 14.

The public language-subtag-registry does not yet have the two new scripts.
I cherry-picked them from their [dec08 registration forms](https://mailarchive.ietf.org/arch/msg/ietf-languages/re5D4wIgAeHUiN8YNLZS61UbTJg/), and set dec22 (+2 weeks) as preliminary addition dates.

Note that some of the tools made some changes in generated files that are not related to the two new scripts.
Apparently these tools have not been run since their code or their input data changed.

Some of these unrelated changes caused test failures, and I manually reverted them.

### CODE REVIEW

**Review by btangmu — DISMISSED** [review]
LGTM, though I'm not familiar with Kawi or Nag Mundari so recommend waiting for more reviewers

**Comment by markusicu** [comment]
Happy new year!
Would anyone familiar enough with script metadata please review this after the holidays?

FYI: The LSR was actually updated with three new script codes on dec24, not dec22. I assume that it's ok that this minor difference from the official data will be erased next time the whole file is ingested.

File: common/supplemental/likelySubtags.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <!--
 Copyright © 1991-2021 Unicode, Inc.
 For terms of use, see http://www.unicode.org/copyright.html
-Unicode and the Unicode Logo are registered trademarks of Unicode, Inc. in the U.S. and other countries.
+SPDX-License-Identifier: Unicode-DFS-2016
```
Comment (srl295) [inline common/supplemental/likelySubtags.xml:6]: +1 expected

**Review by srl295 — APPROVED** [review]
LGTM. I saw this, but wasn't familliar with the U15 scripts themselves.

**Comment by markusicu** [comment]
Thanks, I did the required squash. Please re-approve.

**Review by srl295 — APPROVED** [review]

**Comment by markusicu** [comment]
Thanks @srl295 -- for using these scripts+data in the Unicode Tools, we need to do a mini CLDR release and update some config file in the unicodetools, right?

**Pull request merged**